Tests/MapboxMapsTests/Foundation/MapViewTests.swift

```diff
@@ -257,18 +257,51 @@ final class MapViewTests: XCTestCase {
         XCTAssertEqual(displayLink.$isPaused.setStub.invocations.map(\.parameters), [true])
     }
 
+    func testDisplayLinkPausedWhenAppDidEnterBackground() {
+        notificationCenter.post(name: UIApplication.didEnterBackgroundNotification, object: nil)
+
+        XCTAssertEqual(displayLink.$isPaused.setStub.invocations.map(\.parameters), [true])
```

Comment:
That's that is happening here I believe - checking for the only(and also the last) invocation.

Reply:
My recommendation was more like the next one:
```suggestion
        XCTAssertEqual(displayLink.$isPaused.setStub.invocations.last.parameters, true)
```
So it would be clear that only the last one should be tested and we don't care about other side-effects during the method call.

Reply:
I think by explicitly checking agains a static array we performing more strict testing - this also verifies that there has been one and only invocation to `isPaused` setter.